deploy.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
+set -e
 
-rm -rf .deliver && \
-mkdir .deliver && \
-cp -R www .deliver && \
-echo "Tarring build files" && \
-tar -C .deliver -zcvf .deliver/code_sandbox_client.tar.gz www && \
-echo "Cleaning remote tar" && \
-ssh nginx@ssh.codesandbox.io "rm -f code_sandbox_client.tar.gz" && \
-echo "Moving tar to server" && \
-scp .deliver/code_sandbox_client.tar.gz nginx@ssh.codesandbox.io:./ && \
-echo "Untarring www files" && \
+rm -rf .deliver
+mkdir .deliver
+cp -R www .deliver
+echo "Tarring build files"
+tar -C .deliver -zcvf .deliver/code_sandbox_client.tar.gz www
+echo "Cleaning remote tar"
+ssh nginx@ssh.codesandbox.io "rm -f code_sandbox_client.tar.gz"
+echo "Moving tar to server"
+scp .deliver/code_sandbox_client.tar.gz nginx@ssh.codesandbox.io:./
```

Comment:
Do we need the whole `.deliver` thing? Why not just do
```
tar -zcvf code_sandbox_client.tar.gz www
[...]
scp code_sandbox_client.tar.gz nginx@ssh.codesandbox.io:
```

Reply:
This is a 0 logic change pull aimed at improving maintainability. I don't have enough context to make that change

Reply:
Sorry, the question was more for @CompuIves, i should have mentioned. 😄

Reply:
Hmm, yes. Yesterday we moved to the new deployment flow and this is kind of a leftover. We can just move to `www` directory now 😄 . Let's merge this one in first.